src/ImageRunModal.jsx

```diff
@@ -417,17 +423,20 @@ export class ImageRunModal extends React.Component {
                                      default={{ envKey: null, envValue: null }}
                                      itemcomponent={ <EnvVar />} />
                 </FormGroup>
+                <FormGroup fieldId='run-image-dialog-start-after-creation' label={_("Start after creation")}>
```

Comment:
![Screenshot from 2021-09-03 09-10-15](https://user-images.githubusercontent.com/12330670/131965300-dc367daf-31f6-49df-a988-11fb6aa29e99.png)

It seems misaligned with the label.  If you add `hasNoPaddingTop` prop to this `FormGroup` it should work as expected.